exist-core/src/main/java/org/exist/backup/Backup.java

```diff
@@ -371,13 +369,33 @@ private void backup(final Set<String> seenBlobIds, final Collection current, fin
                     //Skipping resources[i]
                     continue;
                 }
+
                 resource = current.getResource(resources[i]);
 
                 if (dialog != null) {
                     dialog.setResource(resources[i]);
                     dialog.setProgress(i);
                 }
 
+                // Avoid NPE
+                if (resource == null) {
+                    final String msg = "Resource " + resources[i] + " could not be found.";
+                    System.err.println(msg);
+                    Object[] options = {"Ignore", "Abort"};
+                    int n = JOptionPane.showOptionDialog(null, msg,"Backup Error",
```

Comment:
JOptionPane won't work if the Backup is run in console mode instead of GUI mode.